data/macvtap/001-rbac.yaml

```diff
@@ -0,0 +1,50 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: macvtap-cni-cr
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+      - nodes/status
+    verbs:
+      - get
+      - update
```

Comment:
Is this needed? It was for OVS where we use extended resources (manually set the capacity), but with macvtap, the device plugin framework should do that.

Reply:
I sincerely do not know, but the current macvtap-cni repo manifest does not use **any** of this.

I'm assuming I can get rid of it - leaving only the SCC section.